### DISCUSSION TITLE
Feature request: mapping finds on archaeological profiles (in vertical position)

### DISCUSSION THREAD

**Discussion by vogelso** [dainst/idai-field-client#178] [Ideas]
Usually, during archaeological excavations, the profiles are also documented (drawing / photo documentation). Often archaeological finds or samples are also taken from these profiles. Since profiles are always vertical, they cannot be located in a common real-world coordinate system. The idea is to create a container or so within FIELD that allows to map a drawing or a photo documentation of such a profile in a separate coordinate system on which then finds or samples taken from that profile can be mapped.

**Comment by lsteinmann** [comment]
Milestone to keep track: https://github.com/dainst/idai-field/milestone/6
